ingest/src/main/java/com/microsoft/azure/kusto/ingest/ResourceManager.java

```diff
@@ -186,7 +183,7 @@ private void refreshIngestionResources() throws IngestionClientException, Ingest
         }
     }
 
-    private void putIngestionResourceValues(ConcurrentHashMap<ResourceType, IngestionResource> ingestionResources, HashMap<ResourceType, IngestionResource> newIngestionResources) {
+    private void putIngestionResourceValues(ConcurrentHashMap<ResourceType, IngestionResource> ingestionResources, Map<ResourceType, IngestionResource> newIngestionResources) {
```

Comment:
Why make this change again (or is this the reversion)?

Reply:
I don't recall. Though I thought I reverted the entire file, it seems I only reverted the heavy part of the change. This is an improvement that cannot cause harm, so I suggest we leave it.